Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'rummager'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  properties([
+    buildDiscarder(
+      logRotator(
+        numToKeepStr: '50')
+      ),
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    [$class: 'ThrottleJobProperty',
+      categories: [],
+      limitOneJobWithMatchingParams: true,
+      maxConcurrentPerNode: 1,
+      maxConcurrentTotal: 0,
+      paramsToUseForLimit: 'Rummager',
+      throttleEnabled: true,
+      throttleOption: 'category'],
+  ])
+
+  try {
+    stage("Checkout") {
+      checkout scm
+    }
+
+    stage("git merge") {
+      govuk.mergeMasterBranch()
+    }
+
+    stage("bundle install") {
+      govuk.bundleApp()
+    }
+
+    stage("Delete static symlinks and reconnect") {
+      sh '''for d in images javascripts templates stylesheets; do
+              rm -f public/$d
+              ln -s ../../../Static/workspace/public/$d public/
+            done
+         '''
+    }
+
+    stage("rubylinter") {
+      govuk.rubyLinter()
```

Comment:
I was looking into this because I'm adding a Jenkinsfile to some other projects.

The last line determines which directories are analysed by Rubocop. So "spec" will be appropriate for some repos, and "test" will be appropriate for others.

It might be safe to include both, i.e. make the last line `app test spec lib` to cover most (all?) projects. The linting tool handles missing directories gracefully, but only if you use the `--diff` option, which we do. If there are projects with a different structure, maybe we could also modify the puppet script so that it accepts extra directories to include when linting?